src/System.Private.CoreLib/shared/System/AppDomain.cs

```diff
@@ -358,18 +359,18 @@ public ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName, boo
                                                 activationAttributes);
         }
 
-        public ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName, object[] activationAttributes)
+        public ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName, object?[]? activationAttributes)
         {
             return Activator.CreateInstanceFrom(assemblyFile, typeName, activationAttributes);
         }
 
-        public object CreateInstanceFromAndUnwrap(string assemblyFile, string typeName)
+        public object? CreateInstanceFromAndUnwrap(string assemblyFile, string typeName)
         {
             ObjectHandle oh = CreateInstanceFrom(assemblyFile, typeName);
             return oh?.Unwrap(); 
         }
 
-        public object CreateInstanceFromAndUnwrap(string assemblyFile, string typeName, bool ignoreCase, BindingFlags bindingAttr, Binder binder, object[] args, System.Globalization.CultureInfo culture, object[] activationAttributes)
+        public object? CreateInstanceFromAndUnwrap(string assemblyFile, string typeName, bool ignoreCase, BindingFlags bindingAttr, Binder? binder, object?[]? args, System.Globalization.CultureInfo? culture, object?[]? activationAttributes)
```

Comment:
All these `ObjectHandle oh` variables are being guarded with `?` and it seems like CreateInstance can return null but we just haven't annotated those guys. Might want to annotate to avoid revisiting this guys again.

Reply:
Looking at it again I agree it's incongruous that I annotated CreateInstance to be non-nullable but CreateInstanceFromAndUnwrap to be nullable; seems like they should match.